examples/wip/first-person-map/app.js

```diff
@@ -167,7 +176,7 @@ class Root extends Component {
         id: 'trips',
         data: trips,
         getPath: d => d.segments,
-        getColor: d => d.vendor === 0 ? [253, 128, 93] : [23, 184, 190],
+        getColor: d => (d.vendor === 0 ? [253, 128, 93] : [23, 184, 190]),
```

Comment:
Why the parenthesis?

Reply:
just saw these comments, those are from the prettier plugins with no functional changes.
I'll disable prettier for deck.gl before we decide to adopt it.